### PR TITLE
yellowdot 2.4 (new cask)

### PR DESCRIPTION
**Pull request by matchai**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by matchai** [comment]
With `depends_on macos: ">= :monterey"`, I'm surprised it's running on Big Sur. Is this intended?

File: Casks/y/yellowdot.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    version "2.4"
+    sha256 "ba2aeb329e49db187f0954014a8eb5f99f55c078722685fb42e05061209dc908"
+
+    url "https://github.com/FuzzyIdeas/YellowDot/releases/download/v#{version}/YellowDot-#{version}.dmg",
+        verified: "github.com/FuzzyIdeas/YellowDot/"
+
+    livecheck do
+      url :url
+      strategy :github_latest
+    end
```
Comment (khipp) [inline Casks/y/yellowdot.rb:23]:
```suggestion
```

We can use the default `Git` strategy by omitting the `livecheck` block (see [documentation](https://docs.brew.sh/Brew-Livecheck#general-guidelines)).

File: Casks/y/yellowdot.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  end
+
+  name "Yellow Dot"
+  desc "Hide the yellow recording dot in the corner of your screen"
```
Comment (khipp) [inline Casks/y/yellowdot.rb:27]:
```suggestion
  desc "Hides privacy indicators"
```

**Review by khipp — APPROVED** [review]
Thank you, @matchai!

**Pull request merged**